flax/nnx/summary.py

```diff
@@ -340,15 +353,42 @@ def tabulate(
   }
   variable_types: list[type] = sorted(_variable_types, key=lambda t: t.__name__)
 
-  rows: list[CallInfo] = []
-  eval_fn = functools.partial(
-    _call_obj,
-    call_method=method,
-    rows=rows,
-    node_stats=node_stats,
-    object_types=object_types,
-  )
-  nnx.eval_shape(eval_fn, obj, input_args, input_kwargs)
+  # Create a dictionary-version of the object's class. This makes
+  # iteration over methods easier.
+  env = create_obj_env(object_types)
+
+  # Modify all the object's methods to save their Tracer arguments.
+  # tracer_args contains (object, name, args, kwargs) tuples.
+  tracer_args: Tuple[Any, str, Tuple, Dict[str, Any]] = []
```

Comment:
```suggestion
  tracer_args: tuple[tp.Any, str, tuple, dict[str, tp.Any]] = []
```
and similar for other type hints below